### ISSUE TITLE
bug: No results when calling Trouble

### ISSUE THREAD

**Issue by SparrowMike** [folke/trouble.nvim#503]
### Did you check docs and existing issues?

- [X] I have read all the trouble.nvim docs
- [X] I have searched the existing issues of trouble.nvim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

NVIM v0.10.0

### Operating system/version

macOS 14.5 (23F79)

### Describe the bug

After updating the trouble to the latest version I get no results from any of the commands ( see bottom right corner on the image )
![Screenshot 2024-06-12 at 17 30 07](https://github.com/folke/trouble.nvim/assets/69448825/39113c64-ba96-4c1c-8819-49746ef99a0a)

I resolved the issue by installing with commit from 3.0.0 releases, all other releases above it give me the same results, so far I have tried 3.1.0, 3.2.0, 3.3.0 and 3.4.0 - all of them simply make the trouble.nvim unusable for me.

### Steps To Reproduce

1. Follow installation as per documentations
2. Use any trouble keys or direct call Trouble gives me no results warning

### Expected Behavior

No warning, and expected behaviour would be as per documentation

### Repro

```Lua
return {
  {
    "folke/trouble.nvim",
    -- commit = "971ad925ff6fdb57950d7c5d4c5d65e2cdbaecec",
    branch = "main",
    opts = {}, -- for default options, refer to the configuration section for custom setup.
    cmd = "Trouble",
    keys = {
      {
        "<leader>tt",
        "<cmd>Trouble diagnostics toggle<cr>",
        desc = "Diagnostics (Trouble)",
      },
      {
        "<leader>tT",
        "<cmd>Trouble diagnostics toggle filter.buf=0<cr>",
        desc = "Buffer Diagnostics (Trouble)",
      },
      {
        "<leader>cs",
        "<cmd>Trouble symbols toggle focus=false<cr>",
        desc = "Symbols (Trouble)",
      },
      {
        "<leader>cl",
        "<cmd>Trouble lsp toggle focus=false win.position=right<cr>",
        desc = "LSP Definitions / references / ... (Trouble)",
      },
      {
        "<leader>xL",
        "<cmd>Trouble loclist toggle<cr>",
        desc = "Location List (Trouble)",
      },
      {
        "<leader>xQ",
        "<cmd>Trouble qflist toggle<cr>",
        desc = "Quickfix List (Trouble)",
      },
    },
  },
}
```

Please let me know if there is anything else I need to share, here is my full [config](https://github.com/SparrowMike/dotfiles/tree/master/nvim/.config/nvim)

**Comment by folke** [comment]
Can't reproduce. 
If you want me to look into this, then post a minimal repro as is instructed in the issue template

**Comment by SparrowMike** [comment]
Hi @folke, sorry for the poke and sorry for not giving enough to reproduce the issue. 

It is strange - happens to me on my personal machine and my work machine, I have reverted my recent commits to my nvim config just to be sure there wasn't something within my config breaking it.

I have shared the repo to my nvim [configuration](https://github.com/SparrowMike/dotfiles/tree/master/nvim/.config/nvim)

Thanks a lot for all of the amazing work, all works perfectly for me on 3.0.0 release, I will try to create a minimal repo to reproduce the issue.
